t_stress/test_flood.py

```diff
@@ -0,0 +1,141 @@
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2022 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+import socket
+import ssl
+
+import h2
+from h2.settings import SettingCodes
+from hyperframe.frame import HeadersFrame, PriorityFrame
+
+from framework import tester
+from framework.parameterize import param, parameterize
+from helpers import tf_cfg
+
+
+class TestH2ControlFramesFlood(tester.TempestaTest):
+    clients = [
+        {
+            "id": "deproxy",
+            "type": "deproxy_h2",  # "deproxy" for HTTP/1
+            "addr": "${tempesta_ip}",
+            "port": "443",
+            "ssl": True,
+        },
```

Comment:
```suggestion
            "ssl": True,
            "ssl_hostname": "tempesta-tech.com",
        },
```
You use special vhost in these tests